parl/remote/heartbeat_server.py

```diff
@@ -0,0 +1,73 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
```

Comment:
Can we add some tests for this file?

Reply:
We will refactor the code of the heartbeat mechanism recently, so we will add the unitest after the refactoring.